src/components/autocompleter/autocompleter.jsx

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types';
+import React, { forwardRef, useEffect, useState } from 'react';
+import useFetch from '@bloodyaugust/use-fetch';
+import Select from '../select/select.jsx';
+import ListOption from '../list-option/list-option.jsx';
+import mockConstants from '../../mocks/mock-constants.js';
+
+const { API_ROOT } = mockConstants;
+
+const Autocompleter = forwardRef(function Autocompleter(props, ref) {
+  const { execute: fetch } = useFetch();
+  const [models, setModels] = useState(props.models);
+
+  useEffect(() => {
+    fetchModels();
+  }, []);
+
+  function fetchModels(searchString) {
+    fetch(apiEndpoint(searchString))
```

Comment:
nitpick: Since this is a promise, it might be a good idea to `return` it. We aren't doing anything with it now, but I think in general that's a good practice